sdks/python/setup.py

```diff
@@ -203,6 +203,7 @@ def get_version():
     'google-cloud-bigquery>=1.6.0,<1.18.0',
     'google-cloud-core>=0.28.1,<2',
     'google-cloud-bigtable>=0.31.1,<1.1.0',
+    'google-cloud-dlp >=0.12.0,<=0.13.0',
```

Comment:
Version after 0.13 will not support py2. (notice at the top: https://googleapis.dev/python/dlp/latest/gapic/v2/api.html)

I wonder if we need to add a comment note here for the person that will upgrade version ranges next?